feat: Add pyOpenSci as the official mechanism for reviewing scientific python packages (#61)
OpenScienceLabs/opensciencelabs.github.io: content/programs/incubator/index.md

  detailing achievements and future plans.
- Undergo a final review and approval by OSL’s governing board.

> #### Evaluation for Scientific Python Projects via PyOpenSci

Reviewer: ```suggestion
#### Peer review of scientific Python Projects via pyOpenSci
```

> For all Scientific Python projects under the Open Science Labs Incubator

Reviewer: ```suggestion
For all scientific Python projects under the Open Science Labs Incubator
```

there is the Scientific Python project and then the scientific Python broader community. since we are often confused with them i'd suggest using a lower "s" to attempt to clarify this is specific to the Scientific Python project.

program, the evaluation process will be conducted via
[pyOpenSci](https://www.pyopensci.org/), a community-led organization that
offers peer reviews of Python software to enhance its quality, usability, and
long-term maintainability. These peer reviews are pivotal in advancing the
open-source tools that underpin open science initiatives.

#### Benefits of Using pyOpenSci for Evaluation

1. **High-Quality Review**: pyOpenSci's review process focuses on ensuring that
   your Python package meets high standards in terms of code quality,
   documentation, and usability.

> 2. **JOSS Acceptance**: The Journal of Open Source Software (JOSS) recognizes
>    the pyOpenSci peer review, allowing your project to reap the benefits of both
>    pyOpenSci and JOSS through a single review process.

Reviewer: ```suggestion
2. **JOSS Acceptance**: The Journal of Open Source Software (JOSS) accepts
   a pyOpenSci review as their own. As such your project, if it is in scope for JOSS, can reap the benefits of both
   pyOpenSci and JOSS through a single peer review process.
```

Reviewer: just wanted to add the in scope part because if a package isn't in scope for JOSS they won't accept it even if we do.

3. **Community-Led**: Gain insights and recommendations from experts in the open
   science and Python communities.

##### Guidelines and Review Process

> - **Project Guidelines**: All projects should strictly adhere to pyOpenSci's
>   packaging guidelines which can be accessed
>   [here](https://www.pyopensci.org/python-package-guide/).

Reviewer: ```suggestion
- **Project Guidelines**: All projects should follow pyOpenSci's
  packaging guidelines which can be accessed
  [here](https://www.pyopensci.org/python-package-guide/).
```

Just using more relaxed language here because for instance we recommend sphinx but if someone uses mkdocs that is a great platform and they are welcome to use it. so i'd say guidelines. 

id also strongly suggest that you have anyone read our [author guide](https://www.pyopensci.org/software-peer-review/how-to/author-guide.html) prior to submitting paying special attention to the [editor in chief initial checks](https://www.pyopensci.org/software-peer-review/how-to/author-guide.html#does-your-package-meet-packaging-requirements). these checks and this page overviews the process and our specific guidelines for authors so it's an excellent resource to point everyone to first prior to submitting.

- **Peer Review Process**: The detailed guide for pyOpenSci's peer review
  process can be found
  [here](https://www.pyopensci.org/about-peer-review/index.html). We highly
  recommend going through this guide to understand what to expect during the
  review.

> By following pyOpenSci's robust evaluation framework, we aim to fortify the
> quality and impact of the Scientific Python projects emerging from the Open
> Science Labs Incubator program. We encourage all projects to actively engage in
> this evaluation process to facilitate their scientific rigor and reach.

Reviewer: ```suggestion
By following pyOpenSci's robust evaluation framework, we aim to fortify the
quality and impact of the Scientific Python projects emerging from the Open
Science Labs Incubator program. We encourage all projects to actively engage in
this evaluation process to facilitate their scientific rigor and reach.
```

i need to think about this a bit but i wonder if you want to use more relaxed language here?

something like

* the pyOpenSci review process will support improving the quality, usability and maintainability of your software.
* you can also benefit from visibility which your tool will gain once accepted into the pyOpenSci ecosystem and also community support..

something like that? the word evaluation is interesting but i'm not sure it's quite the right word here. and i am not sure about scientific rigor in the above text as we really are generally not focused on the scientific applications rather the software as it's used for science. Although if a tool has a very specific scientific algorithm we'd call in an expert for review. the language just seems a bit different from what we actually do.

Author: hey @lwasser , I hear you and that makes sense. I will update the text with your suggestions. thanks!
